stackslib/src/net/download/nakamoto/tenure_downloader_set.rs

```diff
@@ -472,15 +571,23 @@ impl NakamotoTenureDownloaderSet {
                     &naddr, &downloader.tenure_id_consensus_hash
                 );
                 finished.push(naddr.clone());
-                finished_tenures.push(downloader.tenure_id_consensus_hash.clone());
+                finished_tenures.push(CompletedTenure::from(downloader));
                 continue;
             }
             debug!(
                 "Send request to {} for tenure {} (state {})",
                 &naddr, &downloader.tenure_id_consensus_hash, &downloader.state
             );
             let Ok(sent) = downloader.send_next_download_request(network, neighbor_rpc) else {
-                debug!("Downloader for {} failed; this peer is dead", &naddr);
+                info!(
```

Comment:
This is a nit, but do you think in general we could start putting vars inline where possible as a general coding standard? :D not important by any means but it makes things much easier to read when you see 
```rust
info!("Downloader for tenure {} to {naddr} failed; this peer is dead", downloader.tenure_id_consensus_hash);
```
for example.

Reply:
This made me curious about what improvement we might get if we did this. See https://github.com/stacks-network/stacks-core/pull/5419

Reply:
Sure, I can do that.